src/experiments/logins/schema.json

```diff
@@ -199,6 +199,32 @@
       }
     ],
     "functions": [
+      {
+        "name": "getLoginSavingEnabled",
+        "type": "function",
+        "description": "For the given origin, gets a boolean flag which determines whether login saving is enabled",
```

Comment:
Should also mention that the returned promise is rejected if the URL is malformed.